src/pages/index.tsx

```diff
@@ -73,8 +47,7 @@ async function updateCardContent(id: string, cardDetails: task) {
 
 const Index: FC = () => {
     const { state: appState } = useAppContext();
-    const [filteredTask, setFilteredTask] = useState<any>([]);
-    const { response, error, isLoading } = useFetch(TASKS_URL);
+    const { data: response, isError, isLoading } = useGetAllTasksQuery();
```

Comment:
```suggestion
    const { data: tasks, isError, isLoading } = useGetAllTasksQuery();
```

Reply:
type TasksQueryResponse = {
    message: string;
    tasks: task[];
};
it also has message, that's why I am using response

Reply:
We don't message right? We can remove that in transformResponse.

Reply:
done